avocado/core/jobdata.py

```diff
@@ -41,6 +43,20 @@
 CMDLINE_FILENAME = 'cmdline'
 
 
+def _find_class(module, name):
+    """
+    Look for a class including compatibility workarounds
+    """
+    if module == "avocado.plugins.yaml_to_mux":
+        mod = __import__("avocado_varianter_yaml_to_mux", fromlist=[module])
+        return getattr(mod, name)
+    else:
+        mod = __import__(module)
+        mod = sys.modules[module]
+        klass = getattr(mod, name)
+        return klass
```

Comment:
Yes, my bad. I was looking to the upstream pickle version (https://github.com/python/cpython/blob/master/Lib/pickle.py#L1381-L1392).